pkg/cmd/release/create/create.go

```diff
@@ -1066,15 +1138,40 @@ outer_loop:
 }
 
 func askVersion(ask question.Asker, defaultVersion string) (string, error) {
-	var version string
+	var result string
 	if err := ask(&survey.Input{
 		Default: defaultVersion,
 		Message: "Release Version",
-	}, &version); err != nil {
+	}, &result); err != nil {
+		return "", err
+	}
+	return result, nil
+}
+
+func askVersionMetadata(ask question.Asker, packageId string, packageVersion string) (string, error) {
+	var result string
+	if err := ask(&survey.Input{
+		Default: "",
+		Message: fmt.Sprintf("Using release version %s from package %s. Add +metadata? (blank for none)", packageVersion, packageId),
```

Comment:
TODO: I struggled to come up with a sensible (but not too long) prompt here.
The root problem is that I expect statistically zero people to understand what version metadata is
We should discuss this with the team.

Reply:
Don’t mind the length too much. Got any ideas in how we could make it shorter @slewis74 ?